Makefile

```
@@ -0,0 +1,51 @@
+SHELL=bash
+
+JAVA_OPTS=-Xmx1024m -Xms1024m -Xdebug -Xrunjdwp:transport=dt_socket,address=8004,server=y,suspend=n
+
+WEBSITE_DIR:=${WEBSITE}
+WEBSITE_DEFAULT:="target/website"
```

combining this block with the later `ifndef` one and re-using `WEBSITE` can remove the need for `WEBSITE_DIR` var:

```suggestion
WEBSITE_DEFAULT:="target/website"

ifndef WEBSITE
$(warning WEBSITE env var not found applying default: $(WEBSITE_DEFAULT))
WEBSITE = ${WEBSITE_DEFAULT}
endif
```

then using `WEBSITE` and not `WEBSITE_DIR`